pkg/swiveltesting/dep.go

```diff
@@ -62,6 +64,14 @@ func Deploy(e *Env) (*Dep, error) {
 
 	e.Blockchain.Commit()
 
+	tvAddress, _, tvContract, tvErr := mocks.DeployErc4626(e.Owner.Opts, e.Blockchain)
+
+	if tvErr != nil {
+		return nil, tvErr
+	}
```

Comment:
```suggestion
	erc4626Address, _, erc4626Contract, erc4626Err := mocks.DeployErc4626(e.Owner.Opts, e.Blockchain)

	if erc4626Err != nil {
		return nil, erc4626Err
	}
```
This appears a bit out of pattern. In fact, I think we could also argue that the name for the mocked ERC20 token is not extensible either -- perhaps that should be renamed to `erc20...` so that we have a convention for whenever we want to mock a token standard. Adding suggested changes to follow what we have for `erc20` tokens.

This may also be useful down the road as we deal with ERC5095.

Reply:
these are temp vars and are completely insignificant.

that being said, yeah -- we can use erc20 and erc4626.